ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly-4.18-upgrade-from-stable-4.17.yaml

```diff
@@ -781,6 +781,8 @@ tests:
     cluster_profile: gcp-qe
     env:
       COMPUTE_NODE_TYPE: t2a-standard-4
+      COMPUTE_SERVICE_ACCOUNT: jiwei-worker-sa@openshift-qe.iam.gserviceaccount.com
+      CONTROL_PLANE_SERVICE_ACCOUNT: jiwei-control-plane-sa@openshift-qe.iam.gserviceaccount.com
```

Comment:
Suggest to update `openshift-openshift-tests-private-release-4.18__multi-stable-4.18-upgrade-from-stable-4.17.yaml:- as: gcp-ipi-xpn-mini-perm-amd-fips-f28` together.